### ISSUE TITLE
unexpected handling of labels in add_item

### ISSUE THREAD

**Issue by deanmalmgren** [Doist/todoist-python#17]
I made a [simple command line tool](https://github.com/deanmalmgren/trello-todoist) to make it easy to add overdue trello cards to todoist. I'd particularly like to be able to add labels to my newly added todoist tasks. I implemented something like the following:

```python
response = todoist_api.add_item(
    "add labels to my todoist task cronjob",
    labels=[123, 456],
)
print response
```

which appears to be [the right way to handle this](https://developer.todoist.com/#add-an-item), but the response I get back from the todoist_api indicates otherwise:

```
{u'error_tag': u'INVALID_ARGUMENT_VALUE', u'error_code': 20, u'error_extra': {u'expected': u'list', u'argument': u'labels'}, u'error': u'Invalid argument value'}
```

I was able to fix it by [converting the list of integers into a string representing the list of integers](https://github.com/deanmalmgren/trello-todoist/blob/master/bin/trello-todoist#L146), but this doesn't feel like a particularly pythonic way to handle the list of integers. Before I put together a PR (which I'd be happy to do), are there other parts of the API that are handled in a similar way or is the only case where a string representation of a list is expected to be passed to the API?

**Comment by lefcha** [comment]
You're right, thank you for reporting this.

This should not be a problem for the rest of the methods as the commands/parameters are converted to JSON before being sent to the server, and the `add_item` might be the only exception not handling this correctly.

**Comment by lefcha** [comment]
Closed by 2f8b7106bdb9f543b6bde0022e7045e3bdb2b2bb.
